utils/text_formatting.jsx

```diff
@@ -42,7 +40,7 @@ const cjkPattern = /[\u3000-\u303f\u3040-\u309f\u30a0-\u30ff\uff00-\uff9f\u4e00-
 // - autolinkedUrlSchemes - An array of url schemes that will be allowed for autolinking. Defaults to autolinking with any url scheme.
 // - renderer - a custom renderer object to use in the formatWithRenderer function. Defaults to empty.
 // - minimumHashtagLength - Minimum number of characters in a hashtag. Defaults to 3.
-export function formatText(text, inputOptions) {
+export function formatText(text, inputOptions, emojiMap) {
```

Comment:
The `emojiMap` should probably be part of the `inputOptions` here

Reply:
Since I am out of development time, is it ok to make a continuation ticket for this as well? So I can focus on fixing the rest.

Reply:
Yeah, I can do that. I also need to confirm something with this change since plugins have access to this function, and no matter how we change the function signature, this'll affect their ability to render emojis